acceptance/lib/pxp-agent/bolt_pxp_module_helper.rb

```diff
@@ -50,12 +50,32 @@ def do_module_action(broker, target_identities, module_name, action_name, params
   yield(datas) if block_given?
 end
 
+def run_command(broker, targets, command, metadata, **kwargs, &block)
```

Comment:
As we add more of these helpers, it might be useful to separate out the `run_successful` part as something that invokes a function passed to it and confirms it succeeded, then build `run_successful_*` from it and the associated `run_*` function.

Reply:
This seems like a good path forward for using common methods. 👍